components/ratgdo/number/ratgdo_number.h

```diff
@@ -18,12 +18,15 @@ namespace ratgdo {
     public:
         void dump_config() override;
         void setup() override;
-        void set_number_type(NumberType number_type_);
+        void set_number_type(NumberType number_type);
+        float get_setup_priority() const override { return setup_priority::HARDWARE; }
```

Comment:
This makes sense to me. It would be good to have a comment here as we might forget in a few years

Reply:
Done. Also bumped up setup priority to `HARDWARE+1`.